src/common_modules/vsts-cli-release-common/vsts/cli/vrelease/common/release.py

```diff
@@ -0,0 +1,143 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from webbrowser import open_new
+
+from knack.log import get_logger
+from vsts.release.v4_0.models.release import Release
+from vsts.release.v4_0.models.release_start_metadata import ReleaseStartMetadata
+from vsts.release.v4_0.models.artifact_metadata import ArtifactMetadata
+from vsts.release.v4_0.models.build_version import BuildVersion
+
+from vsts.cli.common.identities import resolve_identity_as_id
+from vsts.cli.common.services import (get_release_client, resolve_instance_and_project)
+from vsts.cli.common.uri import uri_quote
+from .release_definition import get_definition_id_from_name
+
+logger = get_logger(__name__)
+
+def release_create(definition_id=None, definition_name=None, artifact_metadata_list=None, description=None, open_browser=False,
+                team_instance=None, project=None, detect=None):
+    """Request (create) a release.
+    :param definition_id: ID of the definition to create. Required if --name is not supplied.
```

Comment:
for short names like --name to work you will need to give the options list in the arguments.py file. 

similar to this.. ac.argument('team_instance', options_list=('--instance', '-i'))